### ISSUE TITLE
Kedro mlflow CLI does not work inside a subdirectory of the project  for kedro≥0.19.4

### ISSUE THREAD

**Issue by swertz** [Galileo-Galilei/kedro-mlflow#531]
## Description

Running `kedro mlflow ui` doesn't work.

## Context

I have followed the 'Getting Started' guide in https://kedro-mlflow.readthedocs.io/en/stable/source/03_getting_started/index.html but cannot launch any of the MLFlow commands as indicated.

## Steps to Reproduce

Simply install a recent version of Kedro-MLFlow:
```
pip install kedro-mlflow
kedro mlflow -h
```

## Expected Result

The Kedro-MLflow commands should be available.

## Actual Result

```
Usage: kedro [OPTIONS] COMMAND [ARGS]...
Try 'kedro -h' for help.

Error: No such command 'mlflow'.
```

Note that `kedro info` shows that the plugin is available:
```
Installed plugins:
kedro_mlflow: 0.12.1 (entry points:hooks,project)
```

## Your Environment

* `kedro` and `kedro-mlflow` version used (`pip show kedro` and `pip show kedro-mlflow`): 0.19.3 and 0.12.1.
    * I have also tried with kedro <0.19
* Python version used (`python -V`): 3.10.11
* Operating system and version: MacOS 14.3.1

## Does the bug also happen with the last version on master?

Yes, same behaviour.

**Comment by Galileo-Galilei** [comment]
I've tested the following script on Centos & Windows 11: 

```
mamba create -n temp python=3.10 -y
mamba activate temp
pip install kedro==0.19.3 kedro_mlflow
kedro new --starter=spaceflights-pandas
cd spaceflights-pandas
kedro mlflow ui
```

and everything works fine. I can't test on a Mac right now. Can you try to check if it works for you? 

One thing that happen sometimes is that you have a corrupted environment with incompatible packages versions. kedro run  works and it only raises warnings, but I've already noticed that it breaks all CLI discovery. Can you check if you can use another plugin (e.g.  ``kedro-viz``) in your environment?

**Comment by swertz** [comment]
I have just tested again and now everything works fine. Now I'm really confused why it didn't work yesterday... Sorry for the noise!

**Comment by Galileo-Galilei** [comment]
It is very likely that you were not running the command at the root of a kedro project, but this will be possible in 0.19.4!
I don't close the issue since I need to make the change before next release.

**Comment by Galileo-Galilei** [comment]
Closed by #539
